src/Pages/SignUp/component/Phone.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import Theme from '../../../Styles/Theme';
+import { API } from '../../../config';
+
+function Phone({ idata, form, setForm }) {
+  const history = useHistory();
+
+  const handleInput = num => {
+    const { name, value } = num.target;
+    setForm({ ...form, [name]: value });
+  };
```

Comment:
```suggestion
  const handleInput = event => {
    const { name, value } = event.target;
    setForm({ ...form, [name]: value });
  };
```
인자로 넘어오는 것이 숫자가 아니라 이벤트 객체이기 때문에 변수명도 그에 맞게 지어주는게 좋아보입니다